tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -146,23 +147,35 @@ class TestExpectTableRowCountToEqualOtherTable:
     @parameterize_batch_for_data_sources(
         data_source_configs=[PostgreSQLDatasourceTestConfig(), SnowflakeDatasourceTestConfig()],
         data=pd.DataFrame({"a": [1, 2, 3, 4]}),
-        extra_data={"test_table_a": pd.DataFrame({"col_b": ["a", "b", "c", "d"]})},
+        extra_data={"other_table": pd.DataFrame({"col_b": ["a", "b", "c", "d"]})},
     )
-    def test_success(self, batch_for_datasource):
-        expectation = gxe.ExpectTableRowCountToEqualOtherTable(other_table_name="test_table_a")
+    def test_success(
+        self,
+        batch_for_datasource: Batch,
+        extra_table_names_for_datasource: list[str],
```

Comment:
We discussed making this a mapping. I think that's definitely better, but plan to do it in a followup. I want to simplify a bit of the logic around how tables are set up, and that will make converting this to a map simpler.